core/styles/components.css

```diff
@@ -214,3 +214,44 @@
   width: 100%;
   text-align: center;
 }
+
+/* toggle button */
+
+input[type='checkbox'] {
+  position: relative;
+  width: 40px;
+  height: 20px;
```

Comment:
try changing it to `width: 6vh;` and `height: 3vh;`. Should look the same and be responsive.

Reply:
done